paco/concurrent.py

```diff
@@ -92,11 +92,17 @@ class ConcurrentExecutor(object):
 
     Usage::
 
-        pool = ConcurrentExecutor(3)
-        pool.add(coroutine, 'foo', 1)
-        pool.add(coroutine, 'foo', 1)
-        pool.add(coroutine, 'foo', 1)
-        await pool.run(return_exceptions=True)
+        async def sum(x, y):
+            return x + y
+
+        pool = paco.ConcurrentExecutor(limit=2)
+        pool.add(sum, 1, 2)
+        pool.add(sum, None, 'str')
+
+        done, pending = await pool.run(return_exceptions=True)
+        [task.result() for task in done]
+        # => [3, TypeError("unsupported operand type(s) for +: 'NoneType' and 'str'")]
```

Comment:
This line exceeds the maximum limit. We should add `# noqa` at the end of the line, such as:
```python
# => [3, TypeError("unsupported operand type(s) for +: 'NoneType' and 'str'")]  # noqa
```